manim/mobject/text/text_mobject.py

```diff
@@ -752,13 +752,13 @@ def _text2svg(self, color: Color):
         line_spacing /= TEXT2SVG_ADJUSTMENT_FACTOR
 
         dir_name = config.get_dir("text_dir")
-        if not os.path.exists(dir_name):
-            os.makedirs(dir_name)
+        if not Path(dir_name).is_dir():
```

Comment:
Again casts are not important as the `config.get_dir` method already returns a `Path` object

Reply:
Fixed!